### PR TITLE
docs: updated readme

### PR DESCRIPTION
**Pull request by melancholiai**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✖                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✔                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✖                                                                       |

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,79 +7,82 @@
 ![snyk](https://img.shields.io/snyk/vulnerabilities/github/MapColonies/retiler?style=for-the-badge)
 
 ----------------------------------
-retiler is a thin service that supposed to run as a [k8s job](https://kubernetes.io/docs/concepts/workloads/controllers/job/). It will retile a web map service and store the output tiles in a storage.
+retiler is a service intended to run as a [k8s job](https://kubernetes.io/docs/concepts/workloads/controllers/job/).
```
Comment (CptSchnitz) [inline README.md:10]: no reason to mention k8s job, you can just use npm start and it will work exactly the same

File: README.md [hunk 2]
```diff
@@ -7,79 +7,82 @@
...
 
+Tiles for rendering will be consumed from a job queue, currently using [pgboss](https://github.com/timgit/pg-boss).
+Each tile holds a `metatile` value and `ZXY` postition. the tile image will be fetched from an `ArcGIS` service and will be splitted into 256x256 pixel tiles in PNG format according to position and metatile value.
+finally the tiles will be stored on s3 storage.
```
Comment (CptSchnitz) [inline README.md:14]: 1. Capitalize first word in a sentence.
2. The explanation can be expanded a bit more, and be more clear. (for example: why the tiles are splitted, which endpoint is used on arcgis, etc..)
3. A diagram could be nice.

**Review by CptSchnitz — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (melancholiai) [reply README.md]: done.

File: README.md (hunk 2 above)
Comment (melancholiai) [reply README.md]: done.

**Review by CptSchnitz — APPROVED** [review]
Mermaid looks good :)

**Pull request merged**